ivy/functional/frontends/numpy/manipulation_routines/rearranging_elements.py

```diff
@@ -1,6 +1,10 @@
-# local
+# global
```

Comment:
This should be local, since ivy is a local library

Reply:
Thank you for spotting that! I made the change :)